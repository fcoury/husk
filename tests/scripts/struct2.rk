struct Error {
  message: string,
  code: int,
}

fn get_error(code: int) -> Error {
  if code == 0 {
    Error { message: "Success", code: 0 }
  } else {
    Error { message: "Error", code: code }
  }
} 

println(get_error(0));
println(get_error(1));
