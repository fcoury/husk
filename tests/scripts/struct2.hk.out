struct Error {message: String("Success"), code: Int(0)}
struct Error {message: String("Error"), code: Int(1)}
